src/View.js

```diff
@@ -24,6 +24,7 @@ export default class View extends UXBase {
 
 	createZone(zoneConfig) {
 		return new Zone({
+			view: this,
```

Comment:
as a general rule, try to avoid creating circular references like this, e.g. giving a 'child' object a reference to its parent object (`this.expectedZones[0].view === this`). It makes the data structures more difficult to reason about and potentially more fragile. In this case, just pass down the `startMarkName` as a property - that's all the `Zone` needs to know about, and doesn't require a circular reference.